source/documentation/monitoring_apps/metrics.md

```diff
@@ -123,3 +123,84 @@ If you are not receiving any metrics, check the [logs](/monitoring_apps.html#log
 ### More about monitoring
 
 For more information about monitoring apps, see [Monitoring the status of your service](https://www.gov.uk/service-manual/technology/monitoring-the-status-of-your-service) on the Service Manual.
+
+## Backing service metrics
+
+If you use the [PostgreSQL](deploying_services/postgresql/#postgresql), [MySQL](deploying_services/mysql/#mysql) or [Redis](deploying_services/redis/#redis) backing service, you can view metrics on those backing services in the GOV.UK PaaS admin tool.
+
+When you create your backing service instance and bind an app to that backing service, you can access that backing service's metrics for that app. Your backing service plan does not affect which metrics you can view.
+
+You can view metrics up to 60 calendar days before today's date. You can select any date range within this time period.
+
+Backing services are attached to a [space](orgs_spaces_users.html#spaces) within an [org](orgs_spaces_users.html#organisations). You must be able to view a space in the GOV.UK PaaS admin tool to view metrics for backing services in that space.
+
+1. Sign into the GOV.UK PaaS admin tool for [London](https://admin.london.cloud.service.gov.uk/) or [Ireland](https://admin.cloud.service.gov.uk/).
+1. Select the __Organisation__ and __Space__.
+1. Select the __Backing services__ tab and select your backing service.
+1. Select the __Metrics__ tab.
+
+### Metrics definitions - PostgreSQL and MySQL
+
+PostgreSQL and MySQL are relational database management systems.
+
+#### Free disk space
+
+How much hard disk space your database has remaining. Your database will stop working if it runs out of disk space. Upgrade your service plan to get more disk space.
+
+#### CPU utilisation
+
+How much computational work your database is doing. If you think your CPU utilisation is too high, you can optimise your database queries or upgrade your service plan.
+
+#### Open connections
+
+How many open connections there are to your database. Unexpectedly high values may indicate problems with your apps' connection management, or that you need to upgrade your service plan. Unexpectedly low values may indicate the database is unavailable, or that your apps cannot connect to the database.
+
+#### Available Memory
+
+How much memory the virtual machine your database is running on has remaining. Unexpectedly low values may indicate you need to optimise your database queries or upgrade your service plan.
```

Comment:
I'm not 100% sure how query optimisation would affect memory usage. It's maybe better to say "Persistently low values" to communicate that it's OK for it to dip occasionally, but that a long running low figure is maybe bad.

Reply:
agree